bin/rucio-necromancer

```diff
@@ -22,6 +22,7 @@
 # - Benedikt Ziemons <benedikt.ziemons@cern.ch>, 2020
 # - Ocean Monjur <75680423+OCM-7898@users.noreply.github.com>, 2021
```

Comment:
This comment is not for you Joel. I wonder if we should be including author emails such as this ^ in the headers.

Reply:
#5114